web/src/main/java/no/nav/modiapersonoversikt/rest/person/PersonsokController.kt

```diff
@@ -376,12 +376,12 @@ fun PersonsokRequest.tilPdlKriterier(clock: Clock = Clock.systemDefaultZone()):
     }
 
     return listOf(
-        PdlKriterie(PdlFelt.NAVN, navn),
-        PdlKriterie(PdlFelt.ADRESSE, adresse),
-        PdlKriterie(PdlFelt.UTENLANDSK_ID, this.utenlandskID),
-        PdlKriterie(PdlFelt.FODSELSDATO_FRA, fodselsdatoFra),
-        PdlKriterie(PdlFelt.FODSELSDATO_TIL, fodselsdatoTil),
-        PdlKriterie(PdlFelt.KJONN, kjonn)
+        PdlKriterie(PdlFelt.NAVN, navn, searchHistorical = PdlOppslagService.PdlSokeOmfang.HISTORISK_OG_GJELDENDE),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->

**thought:** ser at vi `PdlKriterie` har `boost` (og nå `searchHistorical`), men at regelen for hvordan vi søker (e.g `FUZZY_MATCH` etc) ligger i `PdlFelt`. Så lurer på om vi har rom for å gjøre våres api litt enklere her. 

Hvis vi ser bort fra hva vi har idag så kunne man sett for seg at dette kunne skrives sånn f.eks;
```kotlin
return listOf(
    PdlFelt.NAVN            fuzzyMatch  navn                omfang HISTORISK_OG_GJELDENDE boost 1.5f,
    PdlFelt.UTENLANDSK_ID   isEqualTo   this.utenlandskID   omfang HISTORISK_OG_GJELDENDE,
    PdlFelt.ADRESSE         contains    adresse             omfang GJELDENDE,
    PdlFelt.FODSELSDATO     after       fodselsdatoFra      omfang GJELDENDE,
    PdlFelt.FODSELSDATO     before      fodselsdatoTil      omfang GJELDENDE,
    PdlFelt.KJONN           isEqualTo   kjonn               omfang GJELDENDE,
)
```
(`boost` bare tatt med som eksempel)